ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -164,10 +157,50 @@ def get_winservices_modes(self):
             if svc.monitored():
                 data[svc.id] = {'modes': svc.get_serviceclass_startmodes(),
                                 'mode': svc.startMode,
-                                'monitor': svc.monitored()
+                                'monitor': svc.monitored(),
+                                'severity': svc.getFailSeverity(),
                                 }
         return data
 
+    def getMonitoredDataSource(self):
+        '''Return datasource for template if it exists'''
+        
+        template = self.getRRDTemplate()
+        if template:
+            # look for cached datasource reference
+            if self.datasource_id is not None:
+                datasource = template.datasources._getOb(self.datasource_id, None)
+                return datasource
+            # try DefaultService first
+            datasource = template.datasources._getOb('DefaultService', None)
+            if datasource and self.getMonitored(datasource):
+                self.datasource_id = 'DefaultService'
+                return datasource
+            # check others
+            for datasource in template.getRRDDataSources():
+                if datasource.id == 'DefaultService': continue
+                if hasattr(datasource, 'startmode') and self.getMonitored(datasource):
+                    self.datasource_id = datasource.id
+                    return datasource
+        # if not successful set back to None
+        self.datasource_id = None
+        return None
+
+    def getFailSeverity(self):
+        """
+        Return the severity for this service when it fails.
+        """
+        datasource = self.getMonitoredDataSource()
```

Comment:
here you could get the template, then get the datasource object using self.datasrouce_id